calico_containers/calico_ctl/node.py

```diff
@@ -35,6 +36,8 @@
                             "false" to run in the foreground.  When using
                             libnetwork, this may not be set to "false".
                             [default: true]
+  --runtime=<RUNTIME>       Specify how calico-node processes should be
```

Comment:
"Specify how Calico services..."

I don't want to ossify around `calico-node` terminology.  Let's leave that term just to mean the Docker container image.